anitya/app.py

```diff
@@ -112,13 +112,17 @@ def after_openid_login(resp):
     login was successful on the OpenID server.
     '''
     default = flask.url_for('index')
+    blacklist = APP.config['BLACKLISTED_USERS']
```

Comment:
First up, you probably want to use APP.config.get('BLACKLISTED_USERS', []), to make sure that it is backwards compatible.